src/ctia/stores/es/crud.clj

```diff
@@ -453,13 +453,19 @@ It returns the documents with full hits meta data including the real index in wh
    :props s/Any
    (s/optional-key :sort-extension-definitions) SortExtensionDefinitions})
 
+(defn seven? [{:keys [version]}]
```

Comment:
the name of this function partially reflects what it does ;-)

Reply:
@ereteog I renamed the function and added a docstring